docs/outdoorpvp_template.md

```diff
@@ -1,7 +1,59 @@
-[Database Structure](Database-Structure) > [World-Database](World-Database) > [outdoorpvp_template](outdoorpvp_template)
+# outdoorpvp_template
 
-Column | Type | Description
---- | --- | ---
-TypeId | tinyint(3) unsigned | 
-ScriptName | char(64) | 
-Comment | text | 
+[<-Back-to:World](database-world.md)
+
+<table>
+<tbody>
+<tr class="even">
+<td><p><strong>Field</strong></p></td>
+<td><p><strong>Type</strong></p></td>
+<td><p><strong>Attributes</strong></p></td>
+<td><p><strong>Key</strong></p></td>
+<td><p><strong>Null</strong></p></td>
+<td><p><strong>Default</strong></p></td>
+<td><p><strong>Extra</strong></p></td>
+<td><p><strong>Comment</strong></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#TypeId">TypeId</a></p></td>
+<td><p>tinyint(2)</p></td>
+<td><p>unasigned</p></td>
+<td><p>PRI</p></td>
+<td><p>NO</p></td>
+<td><p></p></td>
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#ScriptName">ScriptName</a></p></td>
+<td><p>char(64)</p></td>
+<td><p>signed</p></td>
+<td><p></p></td>
+<td><p>NO</p></td>
+<td><p>0</p></td>
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#comment">comment</a></p></td>
+<td><p>text</p></td>
+<td><p>signed</p></td>
+<td><p></p></td>
+<td><p>YES</p></td>
+<td><p>NULL</p></td>
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+</tbody>
+</table>
+
+**Description of the fields**
+
+### TypeId
+`field-no-description|1`
```

Comment:
what is that |1 at the end for?

Maybe just add "no description yet - feel free to edit the page"

Reply:
I have no idea, it is something that is used on the TC page.

Reply:
i think it's the number for each column ^^